README.md

```diff
@@ -114,6 +114,64 @@ For example, assuming the following directory bucket name `my-test-bucket--usw2-
 usw2-az1, then the URI used will look like: `s3://my-test-bucket--usw2-az1--x-s3/<PREFIX>` (**please note that the 
 prefix for Amazon S3 Express One Zone should end with '/'**), paired with region us-west-2.
 
+## Distributed checkpoints
+
+### Overview
+
+Amazon S3 Connector for PyTorch provides robust support for PyTorch distributed checkpoints. This feature includes:
+
+- `S3StorageWriter` and `S3StorageReader`: Implementations of PyTorch's StorageWriter and StorageReader interfaces.
+- `S3FileSystem`: An implementation of PyTorch's FileSystemBase.
+
+These tools enable seamless integration of Amazon S3 with 
+[PyTorch Distributed Checkpoints](https://pytorch.org/docs/stable/distributed.checkpoint.html), 
+allowing efficient storage and retrieval of distributed model checkpoints.
+
+### Prerequisites and Installation
+
+PyTorch 2.3 or newer is required.
+To use the distributed checkpoints feature, install S3 Connector for PyTorch with the `dcp` extra:
+
+```sh
+pip install s3torchconnector[dcp]
+```
+
+### Sample Example
+
+End to end examples for using distributed checkpoints with S3 Connector for PyTorch 
```

Comment:
nit: "End to end" in this context is usually hyphenated: "End-to-end" ([source](https://en.wikipedia.org/wiki/End-to-end))